solr/core/src/java/org/apache/solr/core/CoreContainer.java

```diff
@@ -657,6 +658,7 @@ public void load() {
 
     securityConfHandler = isZooKeeperAware() ? new SecurityConfHandlerZk(this) : new SecurityConfHandlerLocal(this);
     reloadSecurityProperties();
+    warnUsersIfSecurityDisabled();
```

Comment:
`warnUsersOnInsecureSettings()` ? Security might be enabled in some way here but we still might prompt with a warning here e.g. auth enabled but TLS disabled. Don't want to be a pain with fixing the method names, but I think this change might make the code more readable.